.pre-commit-hooks.yaml

```diff
@@ -17,10 +17,20 @@
     - shell
 - id: check-cpp-and-cu-unit-test-naming-pattern
   name: Check c++ and cu unit test naming pattern
-  description: Check that all C++ and Cuda unit test files end with `_test.cpp` or `_test.cu`.
-  entry: cpp and cu unit test files must end with _test.cpp
+  description: |-
+    Check that all C++ and Cuda unit test files end with `_test.cpp` or `_test.cu` (no `_tests`)
+    and that they're under a /tests/ folder (not /test/).
```

Comment:
Why not a single line? I will result in better markdown docs since this is the source used for the md docs generation in the readme.